src/components/Settings/SettingTile.tsx

```diff
@@ -18,18 +18,14 @@ export interface SettingOptions {
   loading?: boolean
 }
 
-export interface SettingTileProps {
-  setting: SettingOptions
-}
-
-export const SettingTile: FC<SettingTileProps> = props => {
-  const icon = props.setting.icon ?? null
+export const SettingTile: FC<SettingOptions> = props => {
+  const icon = props.icon ?? null
   const { articleSubtitle } = usePalette()
 
   return (
     <View>
-      {props.setting.title === "Disconnetti" && <Divider />}
-      {props.setting.loading ? (
+      {props.title === "Disconnetti" && <Divider />}
```

Comment:
ok ultimissima cosa giuro, questo è rimasto qui da come funzionava prima, ora che è tutto scritto esplicitamente possiamo toglierlo da qui dentro che è stupido e metterlo direttamente in Settings.tsx insieme al SettingTile

Reply:
👌🏼